sdutil/sdutil.go

```diff
@@ -1,6 +1,11 @@
 package main
 
+import (
+	"github.com/flynn/flynn/pkg/shutdown"
+)
+
 func main() {
+	defer shutdown.Exit()
```

Comment:
We already do signal handling in sdutil and it will be deleted soon, so no need.